app.py

```diff
@@ -1,56 +1,85 @@
 from flask import Flask, render_template, request, session, redirect, url_for
-import session_items as session
+import trello_items as trello
 
-_STATUS = ['Not Started', 'In Progress', 'Completed']
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
```

Comment:
This line is now obsolete (so you can delete this line and `flask_config.py` as well)